docker-bake.hcl

```diff
@@ -203,7 +203,7 @@ target "proofs-tools" {
   dockerfile = "./ops/docker/proofs-tools/Dockerfile"
   context = "."
   args = {
-    CHALLENGER_VERSION="v1.1.0"
+    CHALLENGER_VERSION="v1.1.2"
```

Comment:
For the record, we don't always need this to be an actual challenger release.  It's fine to set it to one of the scheduled publishes from develop to pull in interim changes and then tag the proofs-tools image as a beta.  We can move everything to stable versions before tagging a stable proofs-tools release when we get to the point where that's actually needed. Right now proofs-tools is only used for testing.

But also, this version doesn't exist.  There's a release tag for v1.1.1 pending approval in circleci but I don't see a v1.1.2 at all.

Reply:
v1.1.2 is the optimistic ref that I was going to tag after #12016 merges. Noted. I'll replace this with a git hash.

Reply:
Ah right.  The only catch with a git hash is that we don't publish every commit.  There's a `scheduled-docker-publish` job that runs every hour to publish docker images.

Reply:
Looks like that job can be manually triggered too. Not bad.

Reply:
Fixed. Using the latest scheduled-docker-publish tags - https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/65349/workflows/a29e3fb6-bbc1-47ca-bf69-b3997981da1b/jobs/2708576/steps